packages/manager/apps/pci-kubernetes/src/components/information/ClusterInformation.component.tsx

```diff
@@ -0,0 +1,208 @@
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+  ODS_TILE_VARIANT,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsClipboard,
+  OsdsDivider,
+  OsdsText,
+  OsdsTile,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import { useKube } from '@/api/hooks/useKubernetes';
+import ClusterStatus from './ClusterStatus.component';
+
+export default function ClusterInfos() {
+  const { t } = useTranslation('service');
+  const { t: tDetail } = useTranslation('listing');
+
+  const { kubeId, projectId } = useParams();
+  const { data: kubeDetail, isPending } = useKube(projectId, kubeId);
+
+  const isOvhDefaultGateway = () => {
+    const {
+      privateNetworkRoutingAsDefault,
+      defaultVrackGateway,
+    } = kubeDetail?.privateNetworkConfiguration;
+
+    return !privateNetworkRoutingAsDefault && !defaultVrackGateway;
+  };
+
+  const isVRackDefaultGateway = () => {
+    const { privateNetworkConfiguration } = kubeDetail;
+
+    return (
+      privateNetworkConfiguration &&
+      privateNetworkConfiguration.privateNetworkRoutingAsDefault
+    );
+  };
+
+  const isCustomerDefineVRackGateway = () => {
+    const {
+      privateNetworkRoutingAsDefault,
+      defaultVrackGateway,
+    } = kubeDetail?.privateNetworkConfiguration;
+
+    return !privateNetworkRoutingAsDefault && defaultVrackGateway;
+  };
+
+  if (isPending) {
+    return null;
```

Comment:
return spinner ?

Reply:
In production actually, the page including this Tile is not rendering while the detail fetching is pending, so no need for spinner/skeletons in this case